checks/system/unix.py

```diff
@@ -906,12 +907,15 @@ def _get_subprocess_output(command, log):
     Run the given subprocess command and return it's output. Raise an Exception
     if an error occurs.
     """
-    proc = sp.Popen(command, stdout=sp.PIPE, close_fds=True, stderr=sp.PIPE)
+    stdout_f = tempfile.TemporaryFile('rw')
```

Comment:
Nitpick but could you embed that in a "with" statement so won't have to wait for the garbage collection for the file to be deleted ?